packages/components/chip/src/Chip.stories.tsx

```diff
@@ -29,6 +31,11 @@ export const Default: StoryFn = _args => (
   <div className="flex flex-col items-start gap-md">
     {designs.map(design => (
       <Chip design={design} key={design}>
+        <Chip.LeadingIcon>
```

Comment:
I think that would be better to create a dedicated story for this

Reply:
stories will be fully created on stage 5